pkg/util/grpc/agent_client.go

```diff
@@ -82,11 +81,6 @@ func GetDDAgentSecureClient(ctx context.Context, opts ...grpc.DialOption) (pb.Ag
 }
 
 // getIPCAddressPort returns the host and port for connecting to the main agent
-func getIPCAddressPort() (string, error) {
-	ipcAddress, err := config.GetIPCAddress()
-	if err != nil {
-		return "", err
-	}
-
-	return net.JoinHostPort(ipcAddress, config.Datadog.GetString("cmd_port")), nil
+func getIPCAddressPort(ipcAddress string, cmdPort string) (string, error) {
+	return net.JoinHostPort(ipcAddress, cmdPort), nil
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏This helper function isn't needed anymore